src/cljs/athens/events.cljs

```diff
@@ -291,25 +295,30 @@
 
 
 ;; TODO: dec all indices > closed item
-(reg-event-db
+(reg-event-fx
   :right-sidebar/close-item
   [(interceptors/sentry-span "right-sidebar/close-item")]
-  (fn [db [_ uid]]
-    (let [{:right-sidebar/keys [items]} db]
-      (cond-> (update db :right-sidebar/items dissoc uid)
-        (= 1 (count items)) (assoc :right-sidebar/open false)))))
+  (fn [{:keys [db]} [_ uid]]
+    (let [{:right-sidebar/keys
+           [items]}  db
+          last-item? (= 1 (count items))
+          new-db     (cond-> (update db :right-sidebar/items dissoc uid)
+                       last-item? (assoc :right-sidebar/open false))]
+      {:db       new-db
+       :dispatch [:posthog/report-feature :right-sidebar (not last-item?)]})))
```

Comment:
Why `(not last-item?)`

Reply:
because if it was last item, right sidebar is getting inactive